Tests/Packet++Test/Tests/SipSdpTests.cpp

```diff
@@ -648,6 +648,22 @@ PTF_TEST_CASE(SdpLayerParsingTest)
 } // SdpLayerParsingTest
 
 
+PTF_TEST_CASE(NotSdpLayerParsingTest)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/sip_not_sdp.dat");
+
+	pcpp::Packet notSdpPacket(&rawPacket1);
+
+	PTF_ASSERT_TRUE(notSdpPacket.isPacketOfType(pcpp::SIP));
+	pcpp::SipRequestLayer* sipLayer = notSdpPacket.getLayerOfType<pcpp::SipRequestLayer>();
+	PTF_ASSERT_EQUAL(sipLayer->getContentLength(), 273);
```

Comment:
We can also check that the layer after `SipRequestLayer` is of type `PayloadLayer`